privacy_partner_report/__manifest__.py

```diff
@@ -2,10 +2,11 @@
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 {
     'name': 'Privacy Partner Report',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'category': 'GDPR',
     'summary': 'Show the transactions that a specific partner is involved in.',
     'author': "Eficent, "
+              "initOS GmbH, "
```

Comment:
Please, don't put in 'author', as this is reserved for who created it or has done essential contributions

Reply:
@mreficent Ok, I understand your point and adjusted the manifest as you wished. Nevertheless, I would not say that the migration of a module is not an **essential contribution** no matter how many lines had to be adjusted in the end. 
In general, I would like to suggest adding some Does and Don'ts regarding authorship to the Contribution Guidline as well as the How To Migrate. So you could keep esp. newcomers from falling into this social trap. What do you think?

Reply:
@pedrobaeza What do you think of this suggestion? Should it go in https://odoo-community.org/page/contributing or in https://github.com/OCA/maintainer-tools/wiki migration pages? Could you handle it?

Reply:
First one seems a good place.

Reply:
Could you do it? I am sure you will chose the perfect words.

Reply:
Well, I'm on holidays this week, and next one will be very busy with OpenUpgrade stuff.